PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutVaultListener.kt

```diff
@@ -0,0 +1,26 @@
+package com.paypal.android.paypalwebpayments
+
+import com.paypal.android.corepayments.PayPalSDKError
+
+/**
+ * Implement this callback to receive results from [PayPalWebCheckoutClient].
+ */
+interface PayPalWebCheckoutVaultListener {
```

Comment:
```suggestion
interface PayPalWebVaultListener {
```

Does this need the `Checkout` in the name? Seems like all the methods are just for `Vault`.

Same comment for all of the `PayPalWebCheckoutVault` prefixed classes - seems pretty verbose can we just go with `PayPalWebVault`?

Reply:
Yeah I want to do this too, unfortunately the naming schema includes checkout in `v1`. I'm pretty confident we'll need to start moving towards `v2`. I've had that concern for a while, but this module makes it clear a clear decision.

Reply:
I may actually be down to go for it here. The refactor was interesting, we're browser switching in two different ways in the same PayPal module. `PayPalWebVaultListener` will set us up nicely before addressing existing `v1` naming concerns in `v2`.